rankings/models.py

```diff
@@ -122,6 +128,52 @@ def __str__(self):
         return f'{self.after:.2f} ({delta:.2f})'
 
 
+class Tournament(models.Model):
+    """Tournament model, store the tournament configuration."""
+    active = models.BooleanField(
+        default=True,
+    )
+    name = models.CharField(
+        max_length=255,
+        help_text="Enter the tournament name",
+    )
+    has_round_robin = models.BooleanField(
+        verbose_name="Are there Round Robin rounds?",
+    )
+    no_round_robin_rounds = models.PositiveIntegerField(
+        verbose_name="Number of Round Robin rounds",
+        default=0,
+    )
+    round_robin_finished = models.BooleanField(
+        default=False,
+    )
+    has_elimination = models.BooleanField(
+        verbose_name="Are there Elimination rounds?",
+    )
+    no_players_for_elimination_rounds = models.PositiveIntegerField(
+        verbose_name="Number of players for Elimination rounds",
+        default=0,
+    )
+    games = models.ManyToManyField(
+        Game,
+        related_name="tournaments",
+        blank=True,
+    )
```

Comment:
I would have thought a game would only be associated with a single tournament? In which case the relationship is better defined with a foreign key on the `Game` model.

Reply:
Cheers, good spot :D